cmd/set/smtpConfiguration/smtpConfiguration.go

```diff
@@ -0,0 +1,225 @@
+
+	"github.com/loginradius/lr-cli/cmdutil"
+	"github.com/loginradius/lr-cli/prompt"
+
+	"github.com/spf13/cobra"
+)
+
+
+func NewsmtpConfigurationCmd() *cobra.Command {
+	var provider string
+
+	cmd := &cobra.Command{
+		Use:   "smtp-configuration",
+		Short: "Update the SMTP Configuration",
+		Long:  `Use this command to update the configured SMTP email setting`,
+		Example: heredoc.Doc(`
+		# SMTP Provider's Names we can use in set commands
+		# Mailazy, AmazonSES-USEast, AmazonSES-USWest, AmazonSES-EU, Gmail, 
+		Mandrill, Rackspace-mailgun, SendGrid, Yahoo, Other
+		$ lr set smtp-configuration -p Mailazy
+		? Key: <Key>
+		? Secret: <Secret>
+		? From Name: <Name>
+		? From Email Id: <Email ID>		
+		SMTP settings updated
+			   
+		? Send an email to verify your configuration settings are correct?(Y/N): Yes
+		? To Email : <Email ID for Verification>
+		SMTP settings are verified
+		`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if provider == "" {
+				return &cmdutil.FlagError{Err: errors.New("`provider` is required argument")}
+			}
+			return update(provider)
+		},
+	}
+
+	cmd.Flags().StringVarP(&provider, "provider", "p", "", "Enter the provider name which you want to update.")
+
+	return cmd
+}
+
+func update(provider string) error {
+	var ok bool
+	var num int
+	
+	resp, err := api.GetSMTPConfiguration()
+	if err != nil {
+		return err
+	}
+	var isCustomProvider bool
+	for i, val := range cmdutil.SmtpProviders {
+		if strings.ToLower(val.Display) == strings.ToLower(provider) {
+			ok = true 
+			num = i
+		} 
+		if resp.SmtpHost ==  val.SmtpHost {
+			isCustomProvider = true
+		}
+	}
+	if !ok {
+		return errors.New("SMTP Provider is not found.")
+	} 
+			
+	if resp.FromEmailId != "" && 
+		(strings.ToLower(resp.Provider) == strings.ToLower(cmdutil.SmtpProviders[num].Name) || 
+		resp.SmtpHost ==  cmdutil.SmtpProviders[num].SmtpHost || (num == 9 && !isCustomProvider))  {
+		ok = true
+	} else {
+		ok = false
+	}
+	if !ok {
+		return errors.New("The configuration for the selected provider is not found.")
+	} 
+	var smtpSchema api.SmtpConfigSchema 
+	var smtpLabels = [] string {"Key","Secret","SmtpHost","SmtpPort",
+	"FromName","FromEmailId","UserName","Password","IsSsl"}
+	var isDisplayed bool
+	var isSsl bool
+	
+	for _,val := range smtpLabels {
+		
+		if val == "IsSsl" && strings.ToLower(provider) != "mailazy" {
+			isSsl = cmdutil.SmtpProviders[num].EnableSSL
+			if err := prompt.Confirm(cmdutil.SmtpOptionNames[val] + ":", 
+						&isSsl); err != nil {
+							return err
+			}
+		} else {
+			if strings.ToLower(provider) == "mailazy"   {
+				isDisplayed = ( val != "Password" && val != "IsSsl" && val != "UserName" && val != "SmtpHost" && val != "SmtpPort" )
+			} else if strings.ToLower(provider) != "mailazy" {
+				if num != 9 {
+					isDisplayed = (val != "Key" && val != "Secret" && val != "SmtpHost" && val != "SmtpPort")
+				} else {
+					isDisplayed = (val != "Key" && val != "Secret")
+				}
+			}
```

Comment:
WE can me this as a util function and use it from there

Reply:
Noted